fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -72,6 +72,12 @@ def self.description
         "Create a new Keychain"
       end
 
+      def self.output
+        [
+          ['ORIGINAL_DEFAULT_KEYCHAIN', 'The path to the new system default keychain']
```

Comment:
`ORIGINAL` vs. `new`? Is this correct?

Reply:
Nice catch, according to the comment in the code:

> # if there is no default keychain - setting the original will fail - silent this error

we should change the description to:

```
['ORIGINAL_DEFAULT_KEYCHAIN', 'The path to the default keychain']
```